aldryn_client/localdev/main.py

```diff
@@ -0,0 +1,85 @@
+import os
+import subprocess
+
+import click
+
+
+def create_workspace(client, website_slug, path=None):
+    website_git_path = 'git@github.com:aldryn/test-project.git'  # temporary
```

Comment:
ProTip: if you add a `TODO` to the comment (like `# TODO: ...`) it gets flagged by `pyflakes`, so that we don't loose them :)